src/ColorTypes.jl

```diff
@@ -50,7 +50,7 @@ include("show.jl")
 end # module
 
 if VERSION < v"0.4.0-dev"
-    using Docile
+    using Docile, Compat
```

Comment:
Really, the `using Compat` should be unconditional (not inside an `if` statement).  It is innocuous in 0.4, gives compatibility with some 0.4 pre-releases, and will soon be needed for 0.5 compatibility anyway.